docs/book/src/topics/custom-images.md

```diff
@@ -171,6 +172,53 @@ spec:
           thirdPartyImage: true
 ```
 
+### Using Azure Community Gallery
+
+To use an image from [Azure Community Gallery][azure-community-gallery], set `name` field to gallery's public name and don't set `subscriptionID` and `resourceGroup` fields:
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureMachineTemplate
+metadata:
+  name: capz-community-gallery-example
+spec:
+  template:
+    spec:
+      image:
+        computeGallery:
+          gallery: testGallery-3282f15c-906a-4c4b-b206-eb3c51adb5be
```

Comment:
won't this be confusing for people? sometimes the gallery is uniquename, while other times it is not the unique name, but rather needs subid\rgname. Does it make sense to add another field like "gallery-unique-name" ?

Reply:
From the consumer perspective, it feels to me that it's _just a gallery name_. It should not matter whether it's a private name, resource name or unique name.

Reply:
I am not sure we have it clearly defined in how we write docs but sometime it's nice to use something like `<gallery-name>` , `<name>`, `<version>` so the user doesn't copy paste these values.

Reply:
+1 to using `<gallery-name>` etc to avoid copy paste confusion

Reply:
SGTM

Reply:
Actually, no other code example use placeholders, so I'll leave it as it is.